lib/ext/heroku/command/addons.rb

```diff
@@ -203,7 +203,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      message = ''
+
+      unless addon['config_vars'].empty?
+        message ="Setting #{addon['config_vars'].join(', ')} and restarting #{app}"
+      end
+
+      action(message) do
```

Comment:
Now that this can be an empty string, it makes me wonder if we don't even need the conditional above to be separate from these lines. Does @status get used internally by the heroku-cli tool? I have forgotten.